### ISSUE TITLE
[TypeScript Common] Support schemaNamespace config

### ISSUE THREAD

**Issue by tomitrescak** [dotansimha/graphql-code-generator#1070]
**Describe the bug**

schemaNamespace has no effect on output code

**To Reproduce**

```
overwrite: true
documents:
  - './src/modules/**/*.ts'
  - './src/modules/**/*.tsx'
schema: 'http://localhost:4000/graphql'
generates:
  src/generated/graphql.tsx:
    config:
      noNamespaces: true
      enumsAsTypes: true
    plugins:
      - 'typescript-common'
      - 'typescript-client'
      - 'typescript-react-apollo'
  cypress/fixtures/graphql.d.tsx:
    config:
      enumsAsTypes: true
      schemaNamespace: 'MyNamespace'
    plugins:
      - 'typescript-common'
      - 'typescript-client'
```

**Expected behavior**
the export namespace is present

**Schema/Documents**
If possible and relevant, please provide your GraphQL schema and GraphQL documents.

**Environment:**
 - OS: Mac
 - Codegen: Latest
 - Node: 10

**Comment by dotansimha** [comment]
Hi @tomitrescak !
The `schemaNamespace` is a config for `typescript-server` only.

**Comment by tomitrescak** [comment]
Hi, it's documented in typescript-common. Anyway, this package id already a ser Dreamweaver. Bren using prisma for server and Apollo for client till now but Im so much happier with your work.

If I can add a wish it would be a possibility to generate ambient types wrapped in a namespace.

```
declare namespace MyTypes {
....
```

**Comment by dotansimha** [comment]
Thanks @tomitrescak ! 

It's just a bad copy-paste :) removed from the docs in: https://github.com/dotansimha/graphql-code-generator/commit/379d37bcf842543b5456cf0b9c1f8251d04ee8e9 

We are trying now to find alternatives for namespaces, because not all environments can use it (such as babel-typescript, and it's in use in create-react-app). But I agree that we need to wrap the types of typescript-server with the namespace.

**Comment by tomitrescak** [comment]
Hi, thanks for the fix. The ambient types would be really cool. It solves the nMespacing issues

**Comment by dotansimha** [comment]
Fixed in the recent refactor. we will release it soon.

**Comment by dotansimha** [comment]
Fix available in 1.0.0 👍
